### PR TITLE
Add ephemeral public key to key bundle for UnboxBytes32Any

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: protocol/avdl/crypto.avdl [hunk 1]
```diff
@@ -45,6 +45,7 @@ protocol crypto {
     KID kid;
     EncryptedBytes32 ciphertext;
     BoxNonce nonce;
+    BoxPublicKey ePublicKey;
```
Comment (patrickxb) [inline protocol/avdl/crypto.avdl]: Can we just call it publicKey?  The 'e' doesn't mean much to me, especially three weeks from now.

**Comment by patrickxb** [comment]
LGTM, just one parameter name change suggestion.

File: protocol/avdl/crypto.avdl (hunk 1 above)
Comment (jzila) [inline protocol/avdl/crypto.avdl]: should I call it "ephemeralPublicKey"?

Comment (patrickxb) [inline protocol/avdl/crypto.avdl]: I think publicKey is fine.  The service doesn't really care that it is ephemeral.

**Pull request merged**